docs/usage/self-hosted-configuration.md

```diff
@@ -659,7 +659,7 @@ Instead, with scoped secrets it means that Renovate ensures that the organizatio
 
 <!-- prettier-ignore -->
 !!! note
-    Simple public key encryption was previously used to encrypt secrets, but this approach has been deprecated and is no longer documented.
+    Public key encryption was previously used to encrypt secrets, but this approach has been deprecated and is no longer documented.
```

Comment:
```suggestion
    You could use public key encryption with earlier versions of Renovate.
    We deprecated this approach and removed the documentation for it.
    If you're _still_ using public key encryption then we recommend that you use private keys instead.
```

We can make this text even stronger by:

- using `we` to make it clear _who_ is deprecating, removing stuff and making recommendations
- explaining what a user who's still using public keys should do